sykmeldingalerts-prod.yaml

```diff
@@ -10,28 +10,28 @@ spec:
       channel: '#sykmelding-alerts'
   alerts:
     - alert: syfosm-app-nede
-      expr: kube_deployment_status_replicas_available{deployment=~"syfosmregler|syfosmregister|syfotekster|syfohelsenettproxy|syfosmpapirregler|syfosmmanuell-backend|syfosmmanuell|sykmeldinger-backend|egenmeldt-sykmelding-backend"} == 0
+      expr: kube_deployment_status_replicas_available{deployment=~"syfosmregler|syfoservice-mq-producer|syfosmregister|syfotekster|syfohelsenettproxy|syfosmpapirregler|syfosmmanuell-backend|syfosmmanuell|sykmeldinger-backend|egenmeldt-sykmelding-backend"} == 0
```

Comment:
Samme som for dev: Dette er en asynkron app, den skal ikke være i alerten for de synkrone appene :)